setup.cfg

```diff
@@ -171,3 +171,4 @@ show_traceback = True
 ignore-path = docs/_build
 max-line-length = 80
 sphinx = True
+ignore-path-errors=docs/pages/railway.rst;D000
```

Comment:
Why do we ignore this file?

Reply:
`invalid RST format - D000`

Same root cause of the previous comment, `:force:` directive!!

```rst
.. code:: pycon
  :force:

  >>> assert Failure(1).failure() == 1
  >>> Success(1).failure()
  Traceback (most recent call last):
    ...
  returns.primitives.exceptions.UnwrapFailedError
```

doc8 output:
```text
docs/pages/railway.rst:146: D000 Error in "code" directive:
unknown option: "force".
docs/pages/railway.rst:163: D000 Error in "code" directive:
unknown option: "force".
```

Reply:
The same, this issue should be reported.

Reply:
Seems the issue was already reported: https://github.com/PyCQA/doc8/issues/30